libc/test/src/sys/mman/linux/CMakeLists.txt

```diff
@@ -128,6 +128,22 @@ add_libc_unittest(
     libc.test.UnitTest.ErrnoSetterMatcher
 )
 
+add_libc_unittest(
+  remap_file_pages_test
+  SUITE
+    libc_sys_mman_unittests
+  SRCS
+    remap_file_pages_test.cpp
+  DEPENDS
+    libc.include.sys_mman
```

Comment:
```suggestion
    libc.include.sys_mman
    libc.include.sys_stat
```

Reply:
@SchrodingerZhu  Adding it in the next commit , but can I know why we are adding sys_stat?